ReleaseNotes.md

```diff
@@ -11,6 +11,19 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 08/05/2021 -
+##### Version microsoft/oms:ciprod08052021 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod08052021 (linux)
+##### Code change log
+- Linux Agent
+  - Fix for CPU spike which occurrs at around 6.30am UTC on every day because of unattended package upgrades
+  - Update MDSD build which has fixes for the following issues
+     - Undeterministic Core dump issue because of the non 200 status code and runtime exception stack unwindings
+     - Reduce the verbosity of the error logs for OMS & ODS code paths.
+     - Increase Timeout for OMS Homing service API calls from 30s to 60s
+   - Fix for https://github.com/Azure/AKS/issues/2457
+   - In replicaset, tailing of the mdsd.err log file to agent telemetry
+
```

Comment:
put a note to increase mem request & limit for ds? you can skip specifying values in readme here.

Reply:
Sure, i will update the readme once req & limits in early next week.